HIRS_Utils/src/main/java/hirs/data/persist/certificate/PlatformCredential.java

```diff
@@ -395,7 +401,11 @@ public int getRevisionLevel() {
      * @return the platform specification platform class
      */
     public String getPlatformClass() {
-        return platformClass;
+        if (platformClass != null && !platformClass.isEmpty()) {
```

Comment:
I believe this whole block can be reduced to `return TCG_PLATFORM_MAP.get(platformClass)`. Even if `platformClass` is null or empty. Those keys don't exist in the `TCG_PLATFORM_MAP` and so it will return `null` in those cases, which is what it would do anyway.

Reply:
Agreed.